Open_Source_Events/MWOC/README.md

```diff
@@ -0,0 +1,15 @@
+<h1 align="center"> MEXILI WINTER OF CODE (MWOC) </h1>
+
+## About MWOC
+
+Mexili is a Free and Open Source organization whose motivation is to funnel learning and implementation through Open Source Software Development.
+
+Mexili is an organization that strives towards building a community of enthusiastic and passionate individuals and developers who want to work on building open source projects and learn real software development skills while also targeting to make projects that have a positive impact in the society and help improve productivity alike.
+
+Whether you have an unfinished hackathon project or a completely new idea for an open-source project, at Mexili you can grow and nurture it into a full-fledged product and make an impact towards the community. With an enthusiastic community of developers, Mexili thrives on collaboration and a thriving community of helpful and skilled developers.
+
+<p align="center">
+  <a href="https://mexili.github.io/winter_of_code/#/">
+    <img src="https://raw.githubusercontent.com/smaranjitghose/DocLense/main/readme_assets/mwoc.png" alt="Logo">
```

Comment:
Did you copy this from @smaranjitghose DocLense Project? If yes, please don't.

Reply:
There was no icon or image on the official MWoC website, after searching for a while I found the icon in one of the projects I was contributing to that is DocLense. I will download that image and add it in the codebase rather than using the link, I hope that would be fine? Sorry again for the mistakes.

Reply:
You don't have to be sorry. Can you upload the image on Imgur and reference it there instead of Github Content?

Reply:
Yes I will do that. I have never used Imgur before so please let me know if there are any mistakes in the file.

Reply:
This is weird😅. Just get images from the website or Unsplash. At times this creates an awkward situation for both project maintainers.

Reply:
Yes, I will definitely take care of this in future.